go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/sundayezeilo/post-sql
+
+go 1.22
+
+require (
+	github.com/go-playground/validator v9.31.0+incompatible
```

Comment:
that's a bit odd, what is it for?

Reply:
I'm using the package for request validation

Reply:
I was confused by the org name "go-playground" and was curious if it was some internals of play.go.dev but it seems unrelated... not sure what is there to validate though (I didn't really read their readme)?